types/slonik/slonik-tests.ts

```diff
@@ -110,18 +110,18 @@ const typedQuery = async () => {
   await pool.query(getFooBarQuery(10));
 
   // $ExpectType string
-  await pool.oneFirst(getFooQuery(10));
+  await pool.oneFirst<string>(getFooQuery(10));
```

Comment:
The generic in all these query methods is supposed to represent the row type, which could never be string.